packages/transaction-controller/src/utils/utils.ts

```diff
@@ -167,3 +168,24 @@ export function validateIfTransactionUnapproved(
     );
   }
 }
+
+/**
+ * Checks whether the provided address is strictly equal to the address for
+ * the default swaps token of the provided chain.
+ *
+ * @param address - The string to compare to the default token address
+ * @param chainId - The hex encoded chain ID of the default swaps token to check
+ * @returns Whether the address is the provided chain's default token address
+ */
+export function isSwapsDefaultTokenAddress(address: string, chainId: string) {
```

Comment:
This too would be ideal in `utils/swaps.ts`.

Reply:
Relocated in https://github.com/MetaMask/core/pull/1877/commits/79c777254666aa799f95db6846f4ba579a465f69